pkg/lua/patches/0006-Cleanup-test-module.patch

```diff
@@ -0,0 +1,497 @@
+From 542105cafbba381ef40c414a4025ea25d20371a2 Mon Sep 17 00:00:00 2001
+From: Juan Carrano <j.carrano@fu-berlin.de>
+Date: Mon, 14 May 2018 15:19:14 +0200
+Subject: [PATCH 6/8] Cleanup test module.
+
+This patch makes several improvements to the test module intended
+to allow testing inside embedded environments:
+
+- It does not rely in the standalone intepreter (lua.c).
```

Comment:
Could you describe how to test the changes (even better, provide test snippets)? If, for any of the changes, this is simply something like "the interpreter doesn't crash" or "it compiles" then that's ok too but please describe.

Reply:
Maybe I should add a link to the repo in which I have the modified Lua? I can be downloaded in a desktop and compiled just like normal lua (only better, because mine builds the debug and non-debug versions simultaneously), and can run the test suite just like normal lua.